.travis.yml

```diff
@@ -1,60 +1,60 @@
 sudo: false
 language: node_js
+cache:
+  directories:
+    - node_modules
+addons:
+    code_climate:
+        repo_token: $CODECLIMATE_REPO_TOKEN
```

Comment:
If I understand correctly you rebased this branch on the v2.x branch (which has been merged to master)?  Why are some things marked as additions that are already in the current master? E.g. here ... this should not be new. See:

https://github.com/openpgpjs/openpgpjs/blob/master/.travis.yml#L3-L8